internal/service/cloudprovideraccess/resource_cloud_provider_access_authorization.go

```diff
@@ -260,11 +260,22 @@ func resourceCloudProviderAccessAuthorizationStateUpgradeV0(ctx context.Context,
 }
 
 func authorizeRole(ctx context.Context, client *admin.APIClient, d *schema.ResourceData, projectID string, targetRole *admin.CloudProviderAccessRole) diag.Diagnostics {
-	req := &admin.CloudProviderAccessRole{
+	var roleID string
```

Comment:
knit:
```
roleID := targetRole.GetRoleId()

if targetRole.ProviderName == constant.AZURE {
		roleID = targetRole.GetId()
} 
```

Reply:
after rebasing from the merged master SDK this is no longer relevant